AppInspector.CLI/Writers/AnalyzeHtmlWriter.cs

```diff
@@ -51,9 +52,18 @@ public override void WriteResults(Result result, CLICommandOptions cLICommandOpt
 
         private void WriteHtmlResult()
         {
+            //Grab any local css and js files that are needed i.e. don't have hosted URL's or are proprietary
+            string allCSS = "<style>\n" + MergeResourceFiles(Utils.GetPath(Utils.AppPath.basePath) + "html\\resources\\css");
```

Comment:
You should be using WebPack in the build or pre-build process to pack and minify these assets.

https://www.c-sharpcorner.com/article/getting-started-with-webpack/